deeplinkdispatch-processor/src/test/resources/DeepLinkActivity.java

```diff
@@ -15,6 +15,14 @@
 public class DeepLinkActivity extends Activity {
   private static final String TAG = DeepLinkActivity.class.getSimpleName();
 
+  public static final String ACTION = "com.airbnb.deeplinkdispatch.DEEPLINK_ACTION";
+
+  public static final String EXTRA_RESULT_OK = "com.airbnb.deeplinkdispatch.EXTRA_RESULT_OK";
```

Comment:
`EXTRA_RESULT_OK` was not very clear to me at first. What about `EXTRA_SUCCESSFUL`? Do you think that's clearer?